lib/web_ui/dev/test_platform.dart

```diff
@@ -55,14 +55,16 @@ class BrowserPlatform extends PlatformPlugin {
   ///
   /// [root] is the root directory that the server should serve. It defaults to
   /// the working directory.
-  static Future<BrowserPlatform> start({String root}) async {
+  static Future<BrowserPlatform> start({String root, bool isUpdateScreenshotGoldens: false}) async {
     var server = shelf_io.IOServer(await HttpMultiServer.loopback(0));
     return BrowserPlatform._(
-        server,
-        Configuration.current,
-        p.fromUri(await Isolate.resolvePackageUri(
-            Uri.parse('package:test/src/runner/browser/static/favicon.ico'))),
-        root: root);
+      server,
+      Configuration.current,
+      p.fromUri(await Isolate.resolvePackageUri(
+          Uri.parse('package:test/src/runner/browser/static/favicon.ico'))),
+      root: root,
+      isUpdateScreenshotGoldens: isUpdateScreenshotGoldens,
```

Comment:
I know about the styleguide but... `shouldUpdateScreenshotGoldens`?

Reply:
`should` could mean there's possibility that it won't update anything. How about `doUpdateScreenshotGoldens`?

Reply:
I was trying to limit to [`is`, `has`, `can`, `should`], but yeah, `doUpdateScreenshotGoldens` or even just `updateScreenshotGoldens` sounds a little bit better (to me)

Reply:
+1 to bool updateScreenshotGoldens

Reply:
`updateScreenshotGoldens` sounds too much like a method name. I'll go with `do`. I think anything that sounds like a statement that could be true or false is OK. is, has, can, should, did, was, must, all sound fine to me.

Reply:
I draw the line on archaisms though. `shallWeUpdateScreenshotGoldens` is a no-go :)

Reply:
I know my comment isn't helping, it's only adding to the bike-shedding. But `doUpdateScreenshotGoldens` still sounds like a method name to me.

Reply:
`wouldYouBeSoKindToUpdateScreenshotGoldens`  :trollface: